openstack/blockstorage/extensions/quotasets/urls.go

```diff
@@ -11,3 +11,7 @@ func getURL(c *gophercloud.ServiceClient, projectID string) string {
 func getDefaultsURL(c *gophercloud.ServiceClient, projectID string) string {
 	return c.ServiceURL(resourcePath, projectID, "defaults")
 }
+
+func getDetailURL(c *gophercloud.ServiceClient, projectID string) string {
+	return c.ServiceURL(resourcePath, projectID+"?usage=true")
```

Comment:
Instead of adding the query here, it might be cleaner to add it in the function itself such as what's done here:

https://github.com/gophercloud/gophercloud/blob/de4b788e8f384b917d879d1d788765f1c752d3e9/openstack/networking/v2/extensions/lbaas_v2/loadbalancers/requests.go#L189-L190